src/status_im/common/signals/events.cljs

```diff
@@ -70,10 +69,10 @@
       "messages.new"               (messages.transport/sanitize-messages-and-process-response cofx
                                                                                               event-js
                                                                                               true)
-      "wallet"                     (ethereum.subscriptions/new-wallet-event cofx
-                                                                            (js->clj event-js
-                                                                                     :keywordize-keys
-                                                                                     true))
+      "wallet"                     (rf/dispatch [:wallet/signal-fired
+                                                 (js->clj event-js
```

Comment:
I don't know the shape of wallet signals or how frequently they may arrive, but if they come *frequently enough* it may be better to delay the js->clj transformation. That is, leave the decision up to the re-frame event that will actually deal with the signal. Maybe some signals won't require further processing, this is common in many cases, but I don't know about wallet signals, so I can't comment much.

Reply:
yeah that's a true point, there is lots of unused wallet signals. will do 👍